Mobile version add

saikatgpae opened the pull request:
This PR is created to add the mobile version and desktop version of the famous Behance design created by Cindy Shin.

[Design Link](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 


Please find below the video of link of the project details.
[video demo](https://drive.google.com/file/d/1kWh_0mGH8f8eBQB1Cg7cL8JUbNlyVsH_/view?usp=sharing)

In this project has 2 version, mobile and Desktop.
Used HTML, CSC, Bootstrap and JavaScript.
The speakers details of home page is dynamically uploaded with the help of JavaScript.

saikatgpae committed:
Add header page and Speaker pages.

saikatgpae committed:
Add about.html

saikatgpae committed:
Add mobile pop up menus.

saikatgpae committed:
Add desktop menu.

saikatgpae committed:
Allign main pogram.

saikatgpae committed:
Add linters to remote

saikatgpae committed:
Remove linters error.

saikatgpae committed:
Remove unnecessary image.

saikatgpae committed:
Remove the linters error from about.html.

saikatgpae committed:
Remove linters error from style.css.

zhadier commented on about.html:
```diff
@@ -0,0 +1,96 @@
+</head>
+<body>
+    <div class="full m-0 p-0">
+        <div class="mobile-menu container-fluid pt-5 gap-5 bg-dark">
+            <ul class="menu gap-1 fs-5 p-5">
+                <li><a href="./index.html">Home</a></li>
+                <li><a href="./about.html">About</a></li>
+                <li><a href="#main-program">Program</a></li>
+                <li><a href="#main-program">Join</a></li>
+                <li><a href="#partner">Sponsor</a></li>
+                <li><a href="">News</a></li>
+                <li><a href="./about.html">Contact</a></li>
+            </ul>
+        </div>
```
- [x] Please ensure you also add the top black navbar with the Logout link in both your Home and About pages. It's visible in the Behance design which you can see below

![image](https://user-images.githubusercontent.com/90556221/183039179-eb248589-e469-4743-a4bb-2edfebebd39c.png)

saikatgpae replied:
Added the top nav

zhadier commented on about.html:
```diff
@@ -0,0 +1,96 @@
+            <h5>Creative Comon Korea 2022</h5>
+            <h5>Some Rights Reserved.</h5>
+        </div>
+    </footer>
```
- [x] Please ensure your footer has a black background in the about page like in the Behance design.
> Yours
![image](https://user-images.githubusercontent.com/90556221/183040452-2a1c742c-e329-4f51-9ff6-4f04db4e1c8b.png)

> Behance
![image](https://user-images.githubusercontent.com/90556221/183040530-e7edd462-3c02-44fc-955b-131d15ee1359.png)

- [ ] Additionally, you also need to ensure that the gap between the copyright text and the logo in your footer is similar to the provided design. As you can see above, right now there is too much gap between the two.

zhadier commented on index.html:
```diff
@@ -6,10 +6,176 @@
+                <div class="bar" id="bar2"></div>
+                <div class="bar" id="bar3"></div>
+            </header>
+            <main class="main">
+                <div class="main-content pt-5">
+                    <h2>"Hello Sharing World"</h2>
+                    <H1>CARBON CAPTURE SUMMIT - 2022</H1>
+                </div>
+                
+                <p class="border border-3 border-white">Welcome to the Carbon Capture Summit 2022, where industry leaders will meet in Amsterdam to explore next-generation technologies for carbon capture, utilisation, storage, and transportation to meet net-zero emissions targets.</p>
+                <h2 class="text-dark">2022.11.24(THR) ~ 25(FRI)</h2>
+                
+                <h5>National Museum, Van Der Valk, Amsterdam, Netherlands</h5>
+            </main>
```
- [x] Kindly use section tags since they hold more semantic meaning in the given context. Instead of having a `main` and then `div`'s to structure your website, you should make sure the Headline, Speakers and Program sections are all enclosed in a `section` tag instead so we can infer they are different parts of the same project.

- [x] Please increase the height of your Headline section so it conforms to the spacing guidelines of the Behance design
> Yours
![image](https://user-images.githubusercontent.com/90556221/183042005-fe97c609-0d3f-48d3-929d-e2a73bc14c6a.png)

> Behance
![image](https://user-images.githubusercontent.com/90556221/183042710-abbdb15f-482c-4089-983b-a536105016fb.png)

- [x] Additionally, also ensure the date which I have highlighted above is bold in your design and has the same styling as the Behance design

- [x] Lastly, make sure the paragraph describing the event has a fixed width. Right now it takes up as much space as it can unlike the Behance design which makes it as long as the heading.

saikatgpae replied:
Added section tag.

saikatgpae replied:
Adjusted the height and the width of the paragraph

zhadier commented on index.html:
```diff
@@ -6,10 +6,176 @@
+                <h5>National Museum, Van Der Valk, Amsterdam, Netherlands</h5>
+            </main>
+        </div>
+        <div class="" id="main-program">
```
- [x] As mentioned above, kindly use a `section` tag here

saikatgpae replied:
Used section tag for the same.

zhadier commented on index.html:
```diff
@@ -6,10 +6,176 @@
+            <h3 class="text-white text-center pt-4">Main Program</h3>
+            <h3 class="text-danger text-center">____</h3>
+            <div class="mpcontents">
+                <div class="main-pogram-content justify-content-between" id="lecture">
+                    <a href=""><img class="" src="./image/lecture.png" alt="lecture logo"></a>
+                        <h5 class=" program-info text-danger font-weight-bold w-25">lecture</h5>
+                    <p class="program-details w-50 text-white m-0">Listen the speakers from various countries abouth the message of sharing and opening.</p>
+                </div>
+                <div class="main-pogram-content  justify-content-between" id="cc-exhibition">
+                    <a href=""><img class="" src="./image/cc_exhibition.png" alt="cc-exhibition logo"></a>
+                    <h5 class="program-info text-danger font-weight-bold w-25">CC Exhibition</h5>
+                    <p class="program-details w-50 text-white m-0">Appreciate various creating applying CC liscence of artists, Organize from art center nabi.</p>
+                </div>
```
- [x] Kindly use `article` tags to create both your speaker and program cards since they hold semantic meaning unlike `div`. This will improve the accessibility and SEO score of your website a lot.

saikatgpae replied:
Changed the tag to article.

zhadier commented on index.html:
```diff
@@ -6,10 +6,176 @@
+            </main>
+        </div>
+        <div class="" id="main-program">
+            <h3 class="text-white text-center pt-4">Main Program</h3>
+            <h3 class="text-danger text-center">____</h3>
+            <div class="mpcontents">
```
- [x] Please ensure the horizontal line in your program heading has a little space from the program cards. Right now it does not conform to the design parameters of behance.
![image](https://user-images.githubusercontent.com/90556221/183043920-ca98fbf5-a17e-4c18-a8f7-debe98eee3dc.png)

saikatgpae replied:
Made the necessary changes.

zhadier commented on index.html:
```diff
@@ -6,10 +6,176 @@
+    </div>
+
+    <div class="speakers">
+        <h2 class="text-dark pt-4 text-center" id="speakers">"Featured Speakers"</h2>
```
- [x] Please ensure that your Featured speakers heading is bold and has the same font styling as the Behance design.
![image](https://user-images.githubusercontent.com/90556221/183044149-53566464-ebb4-4db8-a932-910339ef60bc.png)
![image](https://user-images.githubusercontent.com/90556221/183044469-51e00435-bf63-4f35-b428-607408c1673f.png)

saikatgpae replied:
Changed the same as advised.

zhadier commented on index.html:
```diff
@@ -6,10 +6,176 @@
+            </div>
+        </div>                  
+    </div>
+    <div class="partner" id="partner">
+        <h2 class="partners_heading text-center text-white pt-4">Partners</h2>
+        <h3 class="text-danger text-center">____</h3>
+        <div class="parner-logo d-flex justify-content-center justify-content-around flex-wrap p-4 gap-2">
+            <h2 class="text-light">mozilla</h2>
+            <h2 class="fs-2 font-monospace fw-light">Google</h2>
+            <h2 class="fw-bolder fs-1">NAVER</h2>
+            <h2 class="fs-3">daumkkao</h2>
+            <h2 class="">airbnb</h2>
+        </div>
+    </div>
```
- [x] Your partners section seems to disappear for the home page in the desktop version as you can see below. Kindly fix this small visual bug

![image](https://user-images.githubusercontent.com/90556221/183044895-77043360-8a7b-4e42-9604-ebf348a3e351.png)

saikatgpae replied:
Adjusted the same.

zhadier commented on about.html:
```diff
@@ -0,0 +1,96 @@
+                <div class="bar" id="bar2"></div>
+                <div class="bar" id="bar3"></div>
+            </header>
+            <main class="main">
+                <div class="main-content pt-5">
+                    <h2>"Hello Sharing World"</h2>
+                    <H1>CARBON CAPTURE SUMMIT - 2022</H1>
+                </div>
+                <p class="border border-3 border-white mt-5">Carbon capture, utilisation, and storage, or CCUS, is an emerging technology for reducing carbon emissions produced from energy production, heavy industries, and transportation. The captured CO2 can be used as a valuable resource or permanently stored deep underground in geological formations, providing the foundation for carbon removal ...
+                <div class="contact text-center">
+                    <h5>Please contact us via email for any further question</h5>
+                    <h5>About CC Global Summit 2022!</h5>
+                    <h5 id="email" class="email text-dark fs-6"><u>ccsummit2022@ccglobal.org</u></h5>
+                </div>
+            </main>
```
- [ ] As specified in the home page, please use `section` tags for the wrappers of the different sections of your projects.

- [ ] Kindly ensure that your text is aligned center in the About page headline heading and description
> Your Design
![image](https://user-images.githubusercontent.com/90556221/183045772-f18f8940-f1b6-4d89-8e58-aa22f84a964a.png)

> Behance Design
![image](https://user-images.githubusercontent.com/90556221/183046428-ea3e135f-e70b-4dea-b139-ce5559b9365d.png)

- [ ] Additionally, please also ensure you have the correct padding and spacings on the left and right like in the Behance design

saikatgpae replied:
Changed as advised.

zhadier commented on about.html:
```diff
@@ -0,0 +1,96 @@
+        <h5 class="partners_heading text-center text-dark">See the Past Global summits</h5>
+        <h3 class="text-danger text-center">____</h3>
+        <p class="text-center p-4">Take a look at last two CC Global Summits, which took place in Buenos Aires and Warsaw.</p>
+        <img class="p-4 w-100" src="./image/about_2011.png" alt="2011 CC summit icon">
+        <img class="p-4 w-100" src="./image/about_2013.png" alt="2013 CC summit icon">
```
- [x] Please ensure that the images are aligned in a row in the past events section of your project's desktop version. Right now they are in a vertical column as you can see below
> Yours
![image](https://user-images.githubusercontent.com/90556221/183047106-2b298c17-2fd4-41e8-b13d-e20627bbc986.png)
> Behance
![image](https://user-images.githubusercontent.com/90556221/183047167-bb6ea2ce-7240-4658-ab49-59c8e3565747.png)

saikatgpae replied:
Resolved the issue.

zhadier commented on main.js:
```diff
@@ -0,0 +1,33 @@
+});
+
+// activating the dropdown menu
+const dropDown = document.querySelector('.dropdown-arrow');
+dropDown.addEventListener('click', () => {
+  // dispalying the hidden speakers
+  document.querySelector('.speaker-6').classList.toggle('dropdown-dispaly');
+  document.querySelector('.speaker-5').classList.toggle('dropdown-dispaly');
+  document.querySelector('.speaker-4').classList.toggle('dropdown-dispaly');
+
```
- [x] One of the core project requirements is to ensure that the section "Featured speakers" is generated dynamically with JavaScript. Kindly ensure you do not hardcode any HTML in the speakers section and use javascript to dynamically generate your speakers instead. You can use an array to store the objects
``` javascript
const speakerList = [ { name: "Speaker Name",... } ]

window.onload = () => {
  speakerList.map(speaker => {
    // code to build the new HTML element, it can use `createElement()` or just template strings(``).
    // it could attach the new element to a parent inside within the loop
    const speakerContainer = document.getElementById("element-id")
    speakerContainer.appendChild(dynamicElements)
    // OR
    speakerContainer.innerHTML = dynamicElements
    // If used inside a loop the innerHTML method should use `+=` instead of just `=`
    })
   // or accumulate the generate elements in a variable and attach them all after it
}

```
Kindly add this functionality.

![image](https://user-images.githubusercontent.com/90556221/183048364-5fc9b0ee-caa5-42a4-b5b8-32a337dc6cb3.png)

saikatgpae replied:
Added the same.

zhadier reviewed (changes requested):
# Hi @saikatgpae  👋 

Good job so far! 🌤️ 
There are some issues that you still need to work on to go to the next project but you are almost there! 🍓 

<img src="https://media4.giphy.com/media/5wWf7GR2nhgamhRnEuA/giphy.gif"/>

## Highlights
✔️ No linter errors
✔️ Github flow was set up correctly


## Required Changes ♻️
- [ ] Kindly add a more descriptive PR description where you mention what features you added in this project, what project is this about and what requirements you fulfilled.
- [ ] Kindlly record a video presenting the features of the project. Add it's link in your PR description and your ReadMe. It must fulfill the following requirements
      - The maximum length of the video is 5 minutes.
      - The video contains a clear and objective description about the project in professional English. 
      - The topic and purpose of the project are made clear.
      - The video contains a demo of the project, showing the pages in different screen sizes.
      - The video contains a highlight about a piece of the code you find interesting or are proud of making.

- [ ] In the "Acknowledgements" section of your ReadMe, kindly give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question @zhadier so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

saikatgpae committed:
Change as per feedback.

saikatgpae committed:
Remove linters error.

Davidosky007 reviewed (changes requested):
Hi @saikatgpae 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Highlights
* No linter errors 👍 
* Descriptive commit messages 👍 

 ## Required Changes ♻️
## Kindly work on all these changes the previous CR requested as they are required 🙏 
## Kindly add a more descriptive PR description where you mention what features you added to this project, what project is this about and what requirements you fulfilled.

 ## Kindly record a video presenting the features of the project. Add its link in your PR description and your ReadMe. It must fulfil the following requirements
- The maximum length of the video is 5 minutes.
- The video contains a clear and objective description of the project in professional English.
- The topic and purpose of the project are made clear.
- The video contains a demo of the project, showing the pages in different screen sizes.
- The video contains a highlight about a piece of the code you find interesting or are proud of making.

##  In the "Acknowledgements" section of your ReadMe, kindly give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidosky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

saikatgpae commented:
> Hi @saikatgpae 👋,
> 
> Great work on making the changes requested by a previous reviewer 👏 You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.
> 
> ## Highlights
> * No linter errors 👍
> * Descriptive commit messages 👍
> 
> ## Required Changes ♻️
> ## Kindly work on all these changes the previous CR requested as they are required 🙏
> ## Kindly add a more descriptive PR description where you mention what features you added to this project, what project is this about and what requirements you fulfilled.
> ## Kindly record a video presenting the features of the project. Add its link in your PR description and your ReadMe. It must fulfil the following requirements
> * The maximum length of the video is 5 minutes.
> * The video contains a clear and objective description of the project in professional English.
> * The topic and purpose of the project are made clear.
> * The video contains a demo of the project, showing the pages in different screen sizes.
> * The video contains a highlight about a piece of the code you find interesting or are proud of making.
> 
> ## In the "Acknowledgements" section of your ReadMe, kindly give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @Davidosky007 in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Changed the same as instructed.

saikatgpae committed:
Update README.md

Updated the live demo, video description of project details and acknowledgement.

saikatgpae committed:
Merge branch 'main' into mobile-version-add

saikatgpae committed:
Update READ.md with live demo link.

mwanawabangona reviewed (approved):
Hi @saikatgpae  ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  
![](https://camo.githubusercontent.com/d348d0cdaa16b202354d12402e048f52cd4d589b4ec0e55f07d62c2f22753376/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f336f373246634a6d4c7a4964594a646d44652f67697068792e676966)

## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implementing the requested changes !! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

degisew reviewed (commented):
Hi @saikatgpae 
good job so far 👏 

### required changes ♻️ 

kindly it is better to make the body overflow hidden when the popup is visible.
